showcase/tests/integration/components/hds/dropdown/toggle/icon-test.js

```diff
@@ -35,13 +40,25 @@ module('Integration | Component | hds/dropdown/toggle/icon', function (hooks) {
 
   // IMAGE (AVATAR)
 
-  test('if an @imageSrc is declared the image should render in the component', async function (assert) {
+  test('if an @imageSrc is declared and exists the image should render in the component', async function (assert) {
     await render(
       hbs`<Hds::Dropdown::Toggle::Icon @icon="user" @text="user menu" @imageSrc="/assets/images/avatar.png" id="test-toggle-icon" />`
     );
     assert.dom('img').exists();
   });
 
+  test('if an @imageSrc is declared but does not exist, the flight icon should render in the component', async function (assert) {
+    this.set('imageSrc', '/assets/images/avatar.png');
+    await render(
+      hbs`<Hds::Dropdown::Toggle::Icon @icon="user" @text="user menu" @imageSrc={{this.imageSrc}} id="test-toggle-icon" />`
+    );
+    // we load the image dynamically to cover this usecase and also to prevent this test from interminently failing for no obvious reason
```

Comment:
```suggestion
    // we load the image dynamically to cover this usecase and also to prevent this test from intermittently failing for no obvious reason
```